middlewares/login.js

```diff
@@ -13,7 +13,8 @@ module.exports.login = async (req, res) => {
         where: {
             [Sequelize.Op.or]: {
                 email: { [Sequelize.Op.iLike]: username },
-                username: { [Sequelize.Op.iLike]: username }
+                username: { [Sequelize.Op.iLike]: username },
```

Comment:
there's a bug on frontend that only allows you to log in via email, and because of this you actually cannot login with username, fyi

Reply:
Ah yes, since it requires an @ in the login field. But you can still use the username when logging in to OMS JC or Wiki